services/application_test.go

```diff
@@ -6,15 +6,15 @@ import (
 	"syscall"
 	"testing"
 
-	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega"
 	"github.com/smartcontractkit/chainlink/internal/cltest"
 	"github.com/tevino/abool"
 )
 
 func TestServices_ApplicationSignalShutdown(t *testing.T) {
-	RegisterTestingT(t)
-	app, cleanup := cltest.NewApplication()
+	config, cleanup := cltest.NewConfig()
 	defer cleanup()
+	app, _ := cltest.NewApplicationWithConfig(config)
```

Comment:
Shouldn't we also cleanup the `app` or is this handled somewhere else?

Reply:
It should be handled by the SIGTERM signal.